ocs_ci/ocs/constants.py

```diff
@@ -337,6 +337,8 @@
 NON_HOSTED_CLUSTER = "non_hosted"
 
 # provider mode constants
+CLUSTERS_PATH = os.path.join(os.path.expanduser("~"), "clusters")
+
```

Comment:
I think this constant is not used any more.

Reply:
removed